unit_tests/test_app.py

```diff
@@ -77,3 +78,7 @@ def test_get_endpoint_not_found(self, mock_query):
 
         response = self.app.get(self.DEED_ENDPOINT + 'CD3456')
         self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+
+    def test_schema_checks(self):
+
```

Comment:
should this space be here? I don't think one new line breaks pep8 per se but other functions we have implemented always start directly under the definition